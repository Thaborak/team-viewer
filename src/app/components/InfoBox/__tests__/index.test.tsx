import React from 'react';
import { render } from '@testing-library/react';

import { InfoBox } from '..';

describe('<InfoBox  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<InfoBox />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
