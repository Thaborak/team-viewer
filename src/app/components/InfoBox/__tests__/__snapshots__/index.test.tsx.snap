// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoBox  /> should match snapshot 1`] = `
<div
  class=""
/>
`;
